client/jetpack-cloud/sections/sidebar-navigation/purchases.tsx

```diff
@@ -1,11 +1,78 @@
+import { chevronLeft, formatListBulletsRTL, payment, receipt, store, tag } from '@wordpress/icons';
+import { useTranslate } from 'i18n-calypso';
+import page from 'page';
 import NewSidebar from 'calypso/jetpack-cloud/components/sidebar';
+import { itemLinkMatches } from 'calypso/my-sites/sidebar/utils';
+
+const PARTNER_PORTAL_ROOT_PATH = '/partner-portal';
+
+const onClickMenuItem = ( path: string ) => {
+	page.redirect( path );
+};
+
+const isSelected = ( link: string ) => {
+	const pathname = window.location.pathname;
+	return itemLinkMatches( link, pathname );
+};
+
+type ItemProps = {
+	icon: JSX.Element;
+	link: string;
+	title: string;
+};
+const createItem = ( { icon, link, title }: ItemProps ) => ( {
+	icon,
+	path: PARTNER_PORTAL_ROOT_PATH,
+	link,
+	title,
+	onClickMenuItem,
+	isSelected: isSelected( link ),
+} );
 
-// This sidebar is what Jetpack Manage customers will see when they select the
-// Purchases item from the top-level navigation sidebar. It'll display options
-// like Licenses, Invoices, Billing, etc.
-// FIXME: Add menu items
 const PurchasesSidebar = () => {
-	return <NewSidebar path="/partner-portal" menuItems={ [] } />;
+	const translate = useTranslate();
+
+	const menuItems = [
+		createItem( {
+			icon: store,
+			link: `${ PARTNER_PORTAL_ROOT_PATH }/billing`,
```

Comment:
Can we move these URLs to `client/jetpack-cloud/sections/sidebar-navigation/lib/constants.ts`?